Jenkinsfile

```diff
@@ -118,13 +118,46 @@ pipeline {
                 }
             }
         }
-    }
-    post {
-        success {
-            hubotSend(message: 'Build success', status: 'INFO', site: 'slack-pr-app-team')
-        }
-        failure {
-            hubotSend(message: 'Build failed', status: 'ERROR', site: 'slack-pr-app-team')
+        stage('Steps: [ x.x ]') {
```

Comment:
Not completely sure about the x.x approach as opposed to only releasing from master. I would hope the service is simple enough that we can simply steam forward in a straight line ahead without the need for backports and stable branches.

Reply:
But am happy to go ahead with this if it's easiest and/or you think it's the smart thing to do